options/options.go

```diff
@@ -238,6 +260,21 @@ func (terragruntOptions *TerragruntOptions) AppendTerraformCliArgs(argsToAppend
 	terragruntOptions.TerraformCliArgs = append(terragruntOptions.TerraformCliArgs, argsToAppend...)
 }
 
+func (terragruntOptions *TerragruntOptions) GetIncludeConfig() *models.IncludeConfig {
+	var ret *models.IncludeConfig
+	ret = nil
+	if len(terragruntOptions.Context.Includes) > 0 {
+		ret = terragruntOptions.Context.Includes[0]
+	}
+	return ret
```

Comment:
Simplify method to:

```go
if len(terragruntOptions.Context.Includes) > 0 {
  return terragruntOptions.Context.Includes[0]
}
return nil
```

Reply:
I'm trying to limit the number of return statements to a minimum. It's the ongoing war on the principle Dijkstra formulated back in the 70ties I think. 

".. every function, and every block within a function, should have one entry and one exit. "

If the multiple return statements principle is the way of coding for terragrunt this can easily be changed.  

https://hackerchick.com/religious-war-48293-single-vs-multiple/

Reply:
I prefer the functional programming approach: use immutable values whenever possible and return them whenever. Multiple returns are hard to follow with lots of mutation, but straightforward otherwise. IMO, the version with two return statements conveys the intention of the code more clearly and succinctly.